build.gradle

```diff
@@ -5,6 +5,16 @@ plugins {
     id 'com.github.spotbugs' version '4.6.0'
 }
 
+ext.projectName = "Openbanking client"
+ext.projectDescription = "Openbanking client"
```

Comment:
Lets use the description from the github repo please `Java client for using the UK Open Banking API.`.